src/main/kotlin/bsu/cc/views/MainView.kt

```diff
@@ -44,34 +60,66 @@ class MainView : View("Conflict Checker") {
         center {
             vbox {
                 addClass(Styles.content)
-                button("Click me") {
-                    setOnAction {
-                        alert(INFORMATION, "Well done!", "You clicked me!")
+                borderpane {
+                    left {
+                        button("Choose File") {
+                            setOnAction {
+                                if(("" + fileNameField.text().text).trim().length < 1) {
+                                    println("Heck")
+                                    val fileChooser = FileChooser()
+                                    val file = fileChooser.showOpenDialog(null)
+                                    fileNameField.text = file.absolutePath
+                                }
+                                DemoParser.producerTest(fileNameField.text)
+                            }
+                        }
                     }
-                }
-                button("Print Test") {
-                    setOnAction{
-                        DemoParser.producerTest(fileNameField.text)
+
+                    center {
+                        fileNameField = textfield("""src\main\resources\Spring 2019 Validation Report Example.xlsx""")
                     }
-                }
-                val greekLetters = listOf("Alpha","Beta",
-                        "Gamma","Delta","Epsilon").observable()
-                listview(greekLetters) {
-                    selectionModel.selectionMode = SelectionMode.SINGLE
-                }
-                button("Highlight Test") {
-                    setOnAction{
-                        DemoParser.highlightTest(fileNameField.text)
+
+                    bottom {
+                        tableview(conflicts) {
+                            column("Conflict ID", Conflict::conflictIdProp)
+                            column("Class ID", Conflict::classNumberProp)
+                            column("Class Name", Conflict::fullNameProp)
+                            column("Start Time", Conflict::timeProp)
+                        }
                     }
                 }
             }
         }
         bottom {
             hbox {
                 addClass(Styles.footer)
-                label("File Name")
-                fileNameField = textfield("""src\main\resources\Spring 2019 Validation Report Example.xlsx""")
+                borderpane {
+                    right {
+                        button("Export") {
+                            setOnAction {
+                                println("Exporting...")
+                            }
+                        }
+                    }
+                }
             }
         }
     }
+}
+
+class Conflict(id: Int, classNumber: String, fullName: String, time: LocalTime) {
```

Comment:
I think having a separate file for UI mapping objects would be better than cluttering the view files